vectordatasource/transform.py

```diff
@@ -1959,6 +1959,93 @@ def admin_boundaries(ctx):
     return layer
 
 
+def _unicode_len(s):
+    if isinstance(s, str):
+        return len(s.decode('utf-8'))
+    elif isinstance(s, unicode):
+        return len(s)
+    return None
+
+
+def _delete_labels_longer_than(max_label, props):
```

Comment:
is `max_label` = `max_length`, both terms are mentioned here but I think only one is used in practice.

Reply:
Well spotted, thanks! I think that was a typo / thinko. Fixed in a75fd59.